src/Ansatz.jl

```diff
@@ -308,13 +312,11 @@ end
 function truncate!(::MixedCanonical, tn::AbstractAnsatz, bond; threshold, maxdim)
     # move orthogonality center to bond
     mixed_canonize!(tn, bond)
-    return truncate!(NonCanonical(), tn, bond; threshold, maxdim, compute_local_svd=false)
+    return truncate!(NonCanonical(), tn, bond; threshold, maxdim, compute_local_svd=true)
```

Comment:
mmm why is `compute_local_svd` required?

Reply:
Yes, since your implementation of `MixedCanonical` does not have the singular values, you need to do the `svd` to be able to truncate